Ensure node and npm are installed (not WSL)

<!-- ⚠️⚠️ Do Not Delete This! These are comments and will not show up in Preview ⚠️⚠️ -->
<!-- Please read our Rules of Conduct: https://opensource.microsoft.com/codeofconduct/ -->
<!-- Please search existing issues to avoid creating duplicates. -->

**Environment:**
 - OS: Windows 10
 - Extension version: 2.1.5
<!-- The extension version can be found by hovering or selecting "Microsoft Edge Tools for VS Code" in the VS Code Extensions Marketplace panel. -->

**Describe the bug:**
<!-- A clear and concise description of what the bug is -->
Similar to #1145, pop-up upon loading a html file says "Ensure \`node\` and \`npm\` are installed to enable automatically reporting issues in source files pertaining to accessibility, compatibility, security, and more."

The output of the Microsoft Edge Tools extension suggests it can't install the "hint" package:
```
Error loading shared "hint" package
Error: Cannot find module 'hint'
Require stack:
- c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1152:15)
    at Function.resolve (node:internal/modules/helpers:190:19)
    at loadPackage (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:6070)
    at a (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:8713)
    at t.loadWebhint (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:9231)
    at t.Analyzer.initWebhint (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:3292)
    at t.Analyzer.validateTextDocument (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:3966)
    at c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:147226
    at o.invoke (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:35810)
    at i.fire (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:36571) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'c:\\Users\\user1\\.vscode\\extensions\\ms-edgedevtools.vscode-edge-devtools-2.1.5\\node_modules\\vscode-webhint\\dist\\src\\server.js'
  ]
}
Unable to install shared webhint instance Error: spawn EINVAL
    at ChildProcess.spawn (node:internal/child_process:421:11)
    at spawn (node:child_process:799:9)
    at t.run (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:7419)
    at createPackageJson (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:5657)
    at t.updateSharedWebhint (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:8143)
    at async a (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:9004)
    at async t.Analyzer.initWebhint (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:3270)
    at async t.Analyzer.validateTextDocument (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:3955)
    at async c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:147218 {
  errno: -4071,
  code: 'EINVAL',
  syscall: 'spawn'
}
Unable to load shared webhint instance Error: Cannot find module 'hint'
Require stack:
- c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1152:15)
    at Function.resolve (node:internal/modules/helpers:190:19)
    at loadPackage (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:6070)
    at a (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:9055)
    at async t.Analyzer.initWebhint (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:3270)
    at async t.Analyzer.validateTextDocument (c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:3955)
    at async c:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\vscode-webhint\dist\src\server.js:1:147218 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'c:\\Users\\user1\\.vscode\\extensions\\ms-edgedevtools.vscode-edge-devtools-2.1.5\\node_modules\\vscode-webhint\\dist\\src\\server.js'
  ]
}
Installing shared version of "hint"
```

The shared version of "hint" never seems to be installed.
I'm reporting this separately to #2348 as I'm not using WSL, although this comment on that issue seems relevant: https://github.com/microsoft/vscode-edge-devtools/issues/2348#issuecomment-2284221667

**Repro steps:**
<!-- Steps to reproduce the behavior. -->

1. Install NodeJS LTS (v20.17.0). (tried using WinGet and manual MSI download/install)
2. Install Edge Tools extension. (order of 1 & 2 doesn't seem to matter)
3. Restart VS Code
4. Open HTML file
5. Pop-up is displayed in VS Code, and errors thrown in extension output.

**Expected behavior:**
<!-- A clear and concise description of what you expected to happen. -->
A shared version of "hint" is installed correctly and the extension is able to report issues in the opened html file. 

**Additional context:**
<!-- Add any other context or about the problem or screenshots here. -->
Similar to suggestion on #1145, I tried removing `C:\Users\user1\AppData\Roaming\Code\User\globalStorage\ms-edgedevtools.vscode-edge-devtool\node_modules`, however the `\ms-edgedevtools.vscode-edge-devtool` directory was empty. 

Per the errors above, I tried removing the `C:\Users\user1\.vscode\extensions\ms-edgedevtools.vscode-edge-devtools-2.1.5\node_modules\` directory and running `npm i -D`. After doing some work, then restarting VS Code, nothing seems to change - the popup and error still occur. 

If I change to my profile directory `C:\Users\user1\` and run `npm install hint` the extension detects that and uses it as a "shared" instance. But, I'd expect the extension to be properly handling this. 

AB#53620388

--- a/packages/extension-vscode/.vscode/launch.json
+++ b/packages/extension-vscode/.vscode/launch.json
@@ -8,8 +8,8 @@
             "name": "Launch Client",
             "runtimeExecutable": "${execPath}",
             "args": ["--extensionDevelopmentPath=${workspaceRoot}"],
-            "stopOnEntry": false,
             "sourceMaps": true,
+            "autoAttachChildProcesses": true,
             "outFiles": ["${workspaceRoot}/dist/src/**/*.js"],
         },
         {
@@ -17,7 +17,6 @@
             "request": "attach",
             "name": "Attach to Server",
             "address": "localhost",
-            "protocol": "inspector",
             "port": 6009,
             "skipFiles": [
                 "<node_internals>/**/*.js"

--- a/packages/extension-vscode/src/extension.ts
+++ b/packages/extension-vscode/src/extension.ts
@@ -21,7 +21,7 @@ const supportedDocuments = activationEvents.map((event: string) => {
 let client: LanguageClient;
 
 export const activate = (context: ExtensionContext) => {
-    const args = [context.globalStoragePath, 'webhint'];
+    const args = [context.globalStorageUri.fsPath, 'webhint'];
     const module = context.asAbsolutePath('dist/src/server.js');
     const transport = TransportKind.ipc;
 

--- a/packages/extension-vscode/src/utils/process.ts
+++ b/packages/extension-vscode/src/utils/process.ts
@@ -7,7 +7,7 @@ import { spawn, SpawnOptions } from 'child_process';
 /* istanbul ignore next */
 export const run = (command: string, options?: SpawnOptions) => {
     const parts = command.split(' ');
-    const spawnOptions: SpawnOptions = { stdio: 'inherit', windowsHide: true, ...options };
+    const spawnOptions: SpawnOptions = { stdio: 'inherit', shell: true, windowsHide: true, ...options };
     const child = spawn(parts[0], parts.slice(1), spawnOptions);
 
     return new Promise<void>((resolve, reject) => {

--- a/packages/extension-vscode/src/utils/process.ts
+++ b/packages/extension-vscode/src/utils/process.ts
@@ -7,7 +7,7 @@ import { spawn, SpawnOptions } from 'child_process';
 /* istanbul ignore next */
 export const run = (command: string, options?: SpawnOptions) => {
     const parts = command.split(' ');
-    const spawnOptions: SpawnOptions = { stdio: 'inherit', shell: true, windowsHide: true, ...options };
+    const spawnOptions: SpawnOptions = { shell: true, stdio: 'inherit', windowsHide: true, ...options };
     const child = spawn(parts[0], parts.slice(1), spawnOptions);
 
     return new Promise<void>((resolve, reject) => {

